e2e/pluginFixtures.js

```diff
@@ -27,7 +27,7 @@
  */
 
 const { test, expect } = require('./baseFixtures');
-// const { createDomainObjectWithDefaults } = require('./appActions');
+const { createDomainObjectWithDefaults, getUUIDFromURL } = require('./appActions');
```

Comment:
We need to either rename the `getUUIDFromURL` import to `getHashUrlToDomainObject` or remove it entirely (if we are removing `getOrCreateDomainObject` as noted back in August)